content/2.0.html.haml

```diff
@@ -0,0 +1,176 @@
+
+    %dd
+      %div
+        %a{:href => 'http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war'}
+          jenkins.war
+
+    %dt
+      Download native packages
+
+    %dd.release-block-platforms
+      %div
+        %img{:src => "/images/os/win_other.png"}
+          %a.release-block-soft{:href => "http://mirrors.jenkins-ci.org/windows-rc/jenkins-2.0.zip"}
+            Windows
+      %div
+        %img{:src => "/images/os/ubuntu.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/debian-rc/jenkins_2.0_all.deb/"}
+            Ubuntu/Debian
+      %div
+        %img{:src => "/images/os/redhat.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/redhat-rc/jenkins-2.0-1.1.noarch.rpm/"}
+            Red Hat/Fedora
+      %div
+        %img{:src => "/images/os/os_macosx.png"}
+          %a.release-block-soft{:href => "http://mirrors.jenkins-ci.org/osx-rc/jenkins-2.0.pkg"}
+            Mac OS X
+      %div
+        %img{:src => "/images/os/opensuse.png"}
+          %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/opensuse-rc/jenkins-2.0-1.2.noarch.rpm"}
+            openSUSE
+
+
+%h2
+  Highlights
+
+%dl#highlights
+  %dt
+    %strong
+      Pipeline as Code
+    keeps delivery
+    %a{:href => 'https://en.wikipedia.org/wiki/Don%27t_repeat_yourself'}
+      DRY
+  %dd
+    %p
+      The new
+      %a{:href => '/solutions/pipeline'}
+        Pipeline
+      functionality in Jenkins allows you to define configuration as code,
+      which can be checked in and version controlled along with the rest of
+      your project's source code.
+
+    %img{:src => '/images/hello-world-pipeline.png', :title => 'A simple build/test pipeline'}/
+
+    %p
+      Defining your pipeline's configuration as code makes it easier to build a
+      very simple "build and test" pipeline, while enabling more advanced and
+      complex pipelines through the expressive Groovy-based domain specific
+      language.
+
+  %dt
+    Out of the box: Easy to get it right
+  %dd
+    For new users, Jenkins 2.0 starts off with set of recommended plugins to
+    help get you started with the right set of tools to get up and running with
+    Jenkins quickly.
+
+    For the more adventurous users, the Jenkins 2.0 initial setup also allows
+    you to pick and choose exactly the plugins you want to meet your specific
+    needs.
+
+    %a{:href => '/images/getting-started-setup.png', :target => '_blank'}
+      %img{:src => '/images/getting-started-setup.png', :title => 'Getting started with Jenkins 2.0',
+          :width => 550}/
+
+  %dt
+    Improved user interface
+  %dd
+    A number of small, but useful improvements, have been made to tidy up the
+    interfaces you use every day.
+
+
+  %dt
+    Totally backwards compatible
+  %dd
+    Jenkins 2.0 is a drop-in replacement of the Jenkins 1.x series of releases
+    and and fully backward compatible. There is practically no reason
+    %em
+      not
+    to upgrade.
```

Comment:
"There **is** practically no reason to upgrade"? Or "there **will be** practically no reason to upgrade"?

"...when it comes out later this year." — it would be nice to give some vague expectation of the timeframe we're looking at.